t/02-stack.t

```diff
@@ -0,0 +1,106 @@
+use strict;
+use warnings;
+use Test::More;
+use Log::Any qw($log);
+use Log::Any::Adapter qw();
+use Log::Any::Adapter::DERIV;
+use Path::Tiny;
+use Future;
+use JSON::MaybeUTF8 qw(:v1);
+use Clone qw(clone);
+
+subtest 'collapse_future_stack' => sub {
```

Comment:
I ran this subtest locally and found out that both `expected_stack` and `arg_stack` were empty so can we add an example where it is not empty?

Reply:
Sorry ? I tried several times they shouldn't be empty. Let me try again.

Reply:
I printed them both with `Date::Dumper` and both were empty (blank on STDOUT)

Reply:
I don't know what command you used, but I'm sure your result is not correct. You can also check the logic. There are 3 items in the array of sample_stack,and both of them are the cloned data from it. Then `shift` will remove one item from `expected_stack`, then there are 2 items there. 

I guess you didn't add a `-l` when you run `prove` ?